doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -0,0 +1,96 @@
+[id='proc-quarkus-creating-proj_{context}']
+
+
+= Creating the school timetable project
+
+The school timetable project lets you get up and running with {PLANNER} and Quarkus application using Apache Maven and the Quarkus Maven plug-in.
+
+.Procedure
+
+. In a command terminal, enter the following command to verify that Maven is using JDK {JDKVersion} and that the Maven version is {MavenVersion} or higher:
++
+[source]
+----
+mvn --version
+----
+
+. If the preceding command does not return JDK {JDKVersion}, add the path to JDK {JDKVersion} to the PATH environment variable and enter the preceding command again.
+
+. To generate the project, enter one of the following commands:
++
+NOTE: Apple macOS and Microsoft Windows are not supported production environments.
++
+* If you are using Linux or Apple macOS, enter the following command:
++
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.13.1.Final:create \
```

Comment:
[blocker] Could you tell me why do you use here quarkus:quarkus-maven-plugin:1.13.1.Final
1) For 7.11 we provide integration with Red Hat build of Quarkus (RHBQ) 1.11.6.
2) The maven repo you have provided contains only 1.11.6 version

Reply:
I simply copied from the Quarkus getting started project. I;ve updated, but kept Final:create. Is that correct?

Reply:
Updated, please confirm.

Reply:
Fixed